CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 This contains only the most important and/or user-facing changes; for a full changelog, see the commit history.
 
+## [2.0.5](https://github.com/ably/ably-js/tree/2.0.5) (#TODO REPLACE WITH RELEVANT DATE)
```

Comment:
@ttypic on a standup you mentioned you would like to move react hooks to the production-ready status in the next **minor** release. Did you literally mean **minor** as in `ably-js 2.1.0` release, or this `2.0.5` would be fine?

Reply:
I think we should make it `2.1.0` first of all because of https://github.com/ably/ably-js/pull/1762, we introduced new `recoveryKeyStorageName` option, I think in semantic versioning it should be treated as minor change

Reply:
> we should make it 2.1.0 first of all because of https://github.com/ably/ably-js/pull/1762, we introduced new recoveryKeyStorageName option

Oh, true. I remember when I was looking at that PR, I thought the same thing: it would make us bump the minor version in the next release. I didn't go through all the changes for the changelog in this PR, so I initially left it as `2.0.5`. But yeah, 2.1.0 will definitely be the version we go with.

Will close this PR and open a `2.1.0` one.